[2022-04-03 09:05:35,948] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: hello_dag.pprint manual__2022-04-03T14:04:53.379270+00:00 [queued]>
[2022-04-03 09:05:35,956] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: hello_dag.pprint manual__2022-04-03T14:04:53.379270+00:00 [queued]>
[2022-04-03 09:05:35,956] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-04-03 09:05:35,956] {taskinstance.py:1244} INFO - Starting attempt 2 of 2
[2022-04-03 09:05:35,956] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-04-03 09:05:36,106] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): pprint> on 2022-04-03 14:04:53.379270+00:00
[2022-04-03 09:05:36,107] {standard_task_runner.py:52} INFO - Started process 4840 to run task
[2022-04-03 09:05:36,113] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'hello_dag', 'pprint', 'manual__2022-04-03T14:04:53.379270+00:00', '--job-id', '262', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmplb0jqnw3', '--error-file', '/tmp/tmput3muxnk']
[2022-04-03 09:05:36,114] {standard_task_runner.py:77} INFO - Job 262: Subtask pprint
[2022-04-03 09:05:36,800] {logging_mixin.py:109} INFO - Running <TaskInstance: hello_dag.pprint manual__2022-04-03T14:04:53.379270+00:00 [running]> on host LAPTOP-5HJG55H7.localdomain
[2022-04-03 09:05:36,974] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=hello_dag
AIRFLOW_CTX_TASK_ID=pprint
AIRFLOW_CTX_EXECUTION_DATE=2022-04-03T14:04:53.379270+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-03T14:04:53.379270+00:00
[2022-04-03 09:05:36,975] {logging_mixin.py:109} WARNING - --- Logging error ---
[2022-04-03 09:05:36,976] {logging_mixin.py:109} WARNING - Traceback (most recent call last):
[2022-04-03 09:05:36,976] {logging_mixin.py:109} WARNING -   File "/usr/lib/python3.8/logging/__init__.py", line 1081, in emit
    msg = self.format(record)
[2022-04-03 09:05:36,976] {logging_mixin.py:109} WARNING -   File "/usr/lib/python3.8/logging/__init__.py", line 925, in format
    return fmt.format(record)
[2022-04-03 09:05:36,976] {logging_mixin.py:109} WARNING -   File "/usr/lib/python3.8/logging/__init__.py", line 664, in format
    record.message = record.getMessage()
[2022-04-03 09:05:36,976] {logging_mixin.py:109} WARNING -   File "/usr/lib/python3.8/logging/__init__.py", line 369, in getMessage
    msg = msg % self.args
[2022-04-03 09:05:36,976] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/xcom_arg.py", line 86, in __str__
    f"dag_id='{self.operator.dag.dag_id}'",
[2022-04-03 09:05:36,976] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 822, in dag
    raise AirflowException(f'Operator {self} has not been assigned to a DAG yet')
[2022-04-03 09:05:36,976] {logging_mixin.py:109} WARNING - airflow.exceptions.AirflowException: Operator <Task(_PythonDecoratedOperator): print_the_context> has not been assigned to a DAG yet
[2022-04-03 09:05:36,976] {logging_mixin.py:109} WARNING - Call stack:
[2022-04-03 09:05:36,978] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/bin/airflow", line 8, in <module>
    sys.exit(main())
[2022-04-03 09:05:36,978] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/__main__.py", line 48, in main
    args.func(args)
[2022-04-03 09:05:36,978] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
[2022-04-03 09:05:36,978] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
[2022-04-03 09:05:36,978] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
[2022-04-03 09:05:36,978] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 105, in _run_task_by_selected_method
    _run_task_by_local_task_job(args, ti)
[2022-04-03 09:05:36,978] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 163, in _run_task_by_local_task_job
    run_job.run()
[2022-04-03 09:05:36,978] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/jobs/base_job.py", line 246, in run
    self._execute()
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/jobs/local_task_job.py", line 103, in _execute
    self.task_runner.start()
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/task/task_runner/standard_task_runner.py", line 41, in start
    self.process = self._start_by_fork()
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 180, in _run_raw_task
    ti._run_raw_task(
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/operators/python.py", line 175, in execute
    self.log.info("Done. Returned value was: %s", return_value)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/usr/lib/python3.8/logging/__init__.py", line 1434, in info
    self._log(INFO, msg, args, **kwargs)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/usr/lib/python3.8/logging/__init__.py", line 1577, in _log
    self.handle(record)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/usr/lib/python3.8/logging/__init__.py", line 1587, in handle
    self.callHandlers(record)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/usr/lib/python3.8/logging/__init__.py", line 1649, in callHandlers
    hdlr.handle(record)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/usr/lib/python3.8/logging/__init__.py", line 950, in handle
    self.emit(record)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING -   File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/log/file_task_handler.py", line 68, in emit
    self.handler.emit(record)
[2022-04-03 09:05:36,979] {logging_mixin.py:109} WARNING - Message: 'Done. Returned value was: %s'
Arguments: (<airflow.models.xcom_arg.XComArg object at 0x7f027f5da2b0>,)
[2022-04-03 09:05:36,982] {xcom.py:447} ERROR - Could not serialize the XCom value into JSON. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your airflow config.
[2022-04-03 09:05:36,982] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1519, in _execute_task
    self.xcom_push(key=XCOM_RETURN_KEY, value=result)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2145, in xcom_push
    XCom.set(
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/session.py", line 67, in wrapper
    return func(*args, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/xcom.py", line 155, in set
    value=cls.serialize_value(value),
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/xcom.py", line 445, in serialize_value
    return json.dumps(value).encode('UTF-8')
  File "/usr/lib/python3.8/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type XComArg is not JSON serializable
[2022-04-03 09:05:36,989] {taskinstance.py:1272} INFO - Marking task as FAILED. dag_id=hello_dag, task_id=pprint, execution_date=20220403T140453, start_date=20220403T140535, end_date=20220403T140536
[2022-04-03 09:05:37,127] {standard_task_runner.py:89} ERROR - Failed to execute job 262 for task pprint
Traceback (most recent call last):
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/cli/commands/task_command.py", line 180, in _run_raw_task
    ti._run_raw_task(
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 1519, in _execute_task
    self.xcom_push(key=XCOM_RETURN_KEY, value=result)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2145, in xcom_push
    XCom.set(
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/utils/session.py", line 67, in wrapper
    return func(*args, **kwargs)
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/xcom.py", line 155, in set
    value=cls.serialize_value(value),
  File "/home/lowenhard/acceleration/env/lib/python3.8/site-packages/airflow/models/xcom.py", line 445, in serialize_value
    return json.dumps(value).encode('UTF-8')
  File "/usr/lib/python3.8/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type XComArg is not JSON serializable
[2022-04-03 09:05:37,169] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-04-03 09:05:37,327] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
